### PR TITLE
Rubocop

### PR DESCRIPTION
**Pull request by iankronquist**
A work in progress. I started with rubocop -a to make  bunch of automatic fixes, I know some people don't like that, but I am not fixing 240+ string interpolation errors by hand.

### CODE REVIEW

**Comment by armiller** [comment]
@iankronquist nice work!. We usually include a `.rubocop.yml` in our cookbooks to exclude the `metadata.rb` and include the `Berksfile` and `Cheffile`. 

`.rubocop.yml` should look like:
```YAML
AllCops:
  Include:
    - '**/Berksfile'
    - '**/Cheffile'
  Exclude:
    - 'metadata.rb'
```

File: metadata.rb [hunk 1]
```diff
@@ -1,13 +1,13 @@
-name             'osl_application_python'
-maintainer       'Oregon State University'
+name 'osl_application_python'
```
Comment (jordane) [inline metadata.rb]: We don't rubocop this file, we include it in the excludes list in `.rubocop.yml`.

File: providers/celery.rb [hunk 2]
```diff
@@ -70,39 +84,45 @@
...
-  cmds[:celerybeat] = "celerybeat" if new_resource.celerybeat
+  cmds[:celerybeat] = 'celerybeat' if new_resource.celerybeat
   if new_resource.celerycam
     if new_resource.django
-      cmd = "celerycam"
+      new_cmd = 'celerycam'
     else
-      raise "No camera class specified" unless new_resource.camera_class
-      cmd = "celeryev --camera=\"#{new_resource.camera_class}\""
+      fail 'No camera class specified' unless new_resource.camera_class
```
Comment (jordane) [inline providers/celery.rb]: You should provide a more helpful failure message.

File: providers/celery.rb [hunk 3]
```diff
@@ -70,39 +84,45 @@
...
+      new_cmd = "celeryev --camera=\"#{new_resource.camera_class}\""
```
Comment (jordane) [inline providers/celery.rb]: Why set `new_cmd` **after** failing?

**Pull request closed without merging**